ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -369,6 +369,17 @@ objects:
           setup-google-cloud-sdk
         fi
 
+        if [[ "${ENABLE_FIPS}" == true ]]; then
+          oc --insecure-skip-tls-verify --request-timeout=5s get nodes -o jsonpath --template '{range .items[*]}{.metadata.name}{"\n"}{end}' > /tmp/nodes
```

Comment:
5s is not a huge timeout, and we'll fail the CI run if this misses.  Can we raise it to 1m or some such?

Reply:
made it 60s